api/api.go

```diff
@@ -230,6 +235,11 @@ func NewServerRoutes(s *Server) *http.ServeMux {
 	if s.cfg.API.RemoteMode {
```

Comment:
I'd love to get a `log.Info` that notifies a user this server is operating with remote mode enabled? Should probably be in `api.Server.Serve`